saveenv.sh

```diff
@@ -4,10 +4,12 @@
 # the page they are created on. Then a user can source this file to restore those environment 
 # variables if their shell session is reset for some reason.
 
-cat > aks_baseline.env << EOF
+DIR_NAME=$(dirname "$0")
+
+cat > $DIR_NAME/aks_baseline.env << EOF
```

Comment:
Just curious what problem this was solving.  We've used this `saveenv.sh` script here and in the CSE hackathons -- never ran into an issue.  Any pointers as to what benefit this version has?  We'll want to replicate it in the other projects that use this once I know. :)

Reply:
I was in a subdirectory and called it using the relative path `../saveenv.sh` which caused the `aks_baseline.env` files to exist twice -- one (older) in the project root directory, the other one in the subdirectory from which I called `savenenv.sh` using the relative path. With this fix, the location of `aks_baseline.env` should always be well-defined and avoid irritation.

Reply:
Ah, that makes sense.  Following the directions would have never gotten someone into that state.  But I can see that if someone was "off the rails" a bit for whatever reason (good/bad/otherwise), this would help.  Cool.